Sources_v3/Models/ChannelReadModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct ChannelReadModel<ExtraData: ExtraDataTypes> {
```

Comment:
Documentation? ;-)

Reply:
Added in 62b46eb0b0bef360c33b591d1dbceb93446701e2